### PR TITLE
Sc 56 setup initial react project

### PR DESCRIPTION
**Pull request by AstralEik**
The completed initial frontend, contains:
A NextJs frontend application,
Docker code to containerize this frontend application,
A single cypress EtE test, making sure that the front end is running.

### CODE REVIEW

File: docker/frontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:14-alpine
```
Comment (jfho) [inline docker/frontend/Dockerfile]: Why is node version 14 used?

File: frontend/README.md
Comment (Spobendonis) [inline frontend/README.md:1]: This seems like leftover from a template? should probably be moved to our proper README. (note that localhost:3000 is our Grafana and localhost:3001 is for frontend)

File: frontend/tailwind.config.js
Comment (Spobendonis) [inline frontend/tailwind.config.js:1]: Have you defended why you use tailwind?

File: frontend/tsconfig.json
Comment (Spobendonis) [inline frontend/tsconfig.json:1]: Have you defended why you use typescript?

**Review by Spobendonis — COMMENTED** [review]
I have added a few comments, there appears to be some remnants of initializing the project, and lacking documentation (I at least couldn't find any on confluence)

File: frontend/tailwind.config.js
Comment (AstralEik) [reply frontend/tailwind.config.js:1]: All of the reasoning that has been done is written in this confluence page:
https://sipn.atlassian.net/wiki/spaces/PMS/pages/8519681/F001+initial+react+project?atlOrigin=eyJpIjoiY2E4ZjRmNWJmYmZmNGM3N2ExYjQ4YzllYTNjMDcxNjQiLCJwIjoiaiJ9

File: frontend/tsconfig.json
Comment (AstralEik) [reply frontend/tsconfig.json:1]: All of the reasoning that has been done is written in this confluence page:
https://sipn.atlassian.net/wiki/spaces/PMS/pages/8519681/F001+initial+react+project?atlOrigin=eyJpIjoiY2E4ZjRmNWJmYmZmNGM3N2ExYjQ4YzllYTNjMDcxNjQiLCJwIjoiaiJ9

File: docker/frontend/Dockerfile (hunk 1 above)
Comment (AstralEik) [reply docker/frontend/Dockerfile]: Clyde is to blame, updated to 19 as it is the current whole-number version listed in the link

File: frontend/README.md
Comment (AstralEik) [reply frontend/README.md:1]: How to run the front end has been moved to official README, anything else that might be used in the future has been moved to a file UsefulLinks.md in the frontend folder.

**Review by jfho — APPROVED** [review]
Looks good.

**Pull request merged**